virttest/qemu_storage.py

```diff
@@ -462,6 +469,7 @@ def compare_images(self, image1, image2, verbose=True):
                 raise exceptions.TestFail("Compared images differ")
             else:
                 raise exceptions.TestError("Error in image comparison")
+            return rv
```

Comment:
For all function of qemu-img, I think it is better to return process.CmdResult object. It is very useful when we need to check stdout in case.